src/domains/issue/repository/issue.repository.impl.ts

```diff
@@ -34,4 +34,14 @@ export class IssueRepositoryImpl implements IssueRepository {
 
     return issue === null ? null : new IssueDTO(issue);
   }
+
+  async getById(issueId: string): Promise<IssueDTO | null> {
+    const issue: Issue | null = await this.db.issue.findUnique({
```

Comment:
There's no need to add `Issue | null` ts type. Types are already inferred from the return type of the called function. It can also generate problems if the return type changes for some reason and the code brakes, it makes it inflexible.
Just a comment, tell me what you think.

Reply:
I feel more confortable if we know when a change in the flow is performed. And if we need to change type we can do it manually. In addition is easier to know what a method is returning without the need of an IDE. For example here in Github